crates/common/tedge_config/src/tedge_config_cli/tedge_config_defaults.rs

```diff
@@ -90,8 +92,12 @@ pub struct TEdgeConfigDefaults {
 
     /// Default service type
     pub default_service_type: String,
+
     /// Default lock files bool
     pub default_lock_files: Flag,
+
+    /// Default max number of packages
+    pub default_max_packages: Quantity,
```

Comment:
Something is wrong here, but beyond the scope of this PR: this default value is not used by `tedge config get software.plugin.max_packages`. Indeed, since https://github.com/thin-edge/thin-edge.io/pull/1936 has been merged, the tedge config command is using the defaults defined in [`tedge_config/.../new.rs`](https://github.com/thin-edge/thin-edge.io/blob/main/crates/common/tedge_config/src/tedge_config_cli/new.rs)